src/main/java/frc/robot/commands/ClimberCommands/ClimbParts/ClimbAndShoot.java

```diff
@@ -26,10 +26,10 @@ public ClimbAndShoot(Climbsubsystem c, SwerveSubsystem s, ArmSubsystem a, PizzaB
     // Add your commands in the addCommands() call, e.g.
     addCommands (
       new PrintCommand("climb and shoot initialize"),
-      // new SpinToArmAngle(a, 135).withTimeout(1),
-      // Commands.waitUntil(()->a.checkEncoderAngleForClimb()),
-      new MotorDown(c).withTimeout(3).alongWith(new Extend(r)),
-      // new ScoreInTrapStutter(p, a),
+      new SpinToArmAngle(a, 260).withTimeout(1),
+      Commands.waitUntil(()->a.checkEncoderAngleForClimb()),
+      new MotorDown(c).alongWith(new Extend(r).withTimeout(3)),
```

Comment:
parenthesis in the wrong place
new MotorDown(c).alongWith(new Extend(r)).withTimeout(3),